### PR TITLE
logstash 6.0.0 and logstash@5.6 5.6.4 (new formula)

### PR DESCRIPTION
**Pull request by jasontedor**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by jasontedor** [comment]
Please note the logstash@5.6 formula in this PR is simply a rename of the existing formula to 5.6; as that formula is already accepted into this repository, logstash@5.6 should not need a review here.

**Review by ilovezfs — CHANGES_REQUESTED** [review]
Please make the addition of logstash@5.6 a separate commit with the commit subject `logstash@5.6 5.6.4 (new formula)`

**Comment by ilovezfs** [comment]
The versioned formula needs to be `keg_only :versioned_formula`

**Comment by jasontedor** [comment]
This is split into two commits now as requested.

File: Formula/logstash@5.6.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
+class LogstashAT56 < Formula
+  desc "Tool for managing events and logs"
+  homepage "https://www.elastic.co/products/logstash"
+  url "https://artifacts.elastic.co/downloads/logstash/logstash-5.6.4.tar.gz"
+  sha256 "62067531ae5ea4878f84b68625cefcbe427c88d4975702d61cb3eecd80687b09"
+  head "https://github.com/elastic/logstash.git"
+
+  bottle :unneeded
```
Comment (ilovezfs) [inline Formula/logstash@5.6.rb]: blank line under this please

Comment (jasontedor) [reply Formula/logstash@5.6.rb]: I pushed.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks again @jasontedor! :fireworks:

**Comment by jasontedor** [comment]
Awesome, thanks. 🎉

**Pull request merged**